pkg/osv/v1/osv.go

```diff
@@ -121,6 +121,7 @@ func checkResponseError(resp *http.Response) error {
 	return fmt.Errorf("server response error: %s", string(respBuf))
```

Comment:
Can't make a comment on the actual lines, so I'll leave it here:

nit: Can we make `MaxQueriesPerRequest = 1000` private? That seems like an internal implementaton detail and it's unused elsewhere.